### DISCUSSION TITLE
Is there a way to hide a specific category in the dashboard

### DISCUSSION THREAD

**Discussion by Dabako** [firefly-iii/firefly-iii#5535] [Firefly III]
Good day everyone,

I have searched a bit and have not found anything about it. I want to hide the category rent in my dashboard overview of the categories, because it is a lot higher than the rest of my categories. Is there a way to achieve that? 

It would make it a lot easier to get a quick overview about my expenses and the difference between them, that would help me in my starting phase before I am really able to set budgets that are split up between different things.

**Comment by JC5** [comment]
There is no way to hide a category.
